apps/CA.pl.in

```diff
@@ -123,6 +123,10 @@ if ($WHAT eq '-newcert' ) {
     # create a certificate
     $RET = run("$REQ -new -x509 -keyout $NEWKEY -out $NEWCERT $DAYS $EXTRA{req}");
     print "Cert is in $NEWCERT, private key is in $NEWKEY\n" if $RET == 0;
+} elsif ($WHAT eq '-newprecert' ) {
```

Comment:
why is this newprecert and the CLI flag is precert?  Is it a combo of new&precert?
Can we do that more cleanly?  LIke just say -precert implies new?